OPI_RELEASE_APPROACH.md

```diff
@@ -0,0 +1,42 @@
+# OPI Release Approach
+
+OPI will eventually release some artifacts around the work being done in the
+project. What those artifacts are is still being discussed. However, *how*
+they are released, and how it relates to the form the project takes, is
+something worth documenting. This document is the result of a
+[mailing list discussion](https://lists.opiproject.org/g/tsc/message/30)
+around the OPI release process.
+
+## Proposed Release Approach
+
+This document proposes OPI will start by releasing an API in the form of
+protobuf specs and a reference architecture not meant to be used in
+production. The details of what this will look like are as follows:
+
+* The protobuf files themselves.
+  * Language specific bindings for those files. For example, bindings for
+    C++, golang, python, and Rust.
+  * Documentation around the protobuf files
+* A reference implementation which implements the protobuf files
+  * gRPC server code which incorporates the protobuf files.
+  * Implementation specific examples of the APIs themselves.
```

Comment:
what about OTEL and SZTP ? we also producing code there

Reply:
Good point.